run_cgw.sh

```diff
@@ -17,37 +17,43 @@ DEFAULT_DB_PORT=5432
 DEFAULT_REDIS_DB_IP="127.0.0.1"
 DEFAULT_REDIS_DB_PORT=6379
 
+LOG_LEVEL="${CGW_LOG_LEVEL:-info}"
 ID="${CGW_ID:-1}"
-
-GRPC_IP="${CGW_GRPC_IP:-$DEFAULT_SRC_IP}"
-GRPC_PORT="${CGW_GRPC_PORT:-$DEFAULT_GRPC_PORT}"
 WSS_LOCAL_IP="${CGW_WSS_IP:-$DEFAULT_WSS_IP}"
 WSS_LOCAL_PORT="${CGW_WSS_PORT:-$DEFAULT_WSS_PORT}"
+GRPC_IP="${CGW_GRPC_IP:-$DEFAULT_SRC_IP}"
+GRPC_PORT="${CGW_GRPC_PORT:-$DEFAULT_GRPC_PORT}"
 KAFKA_IP="${CGW_KAFKA_IP:-$DEFAULT_KAFKA_IP}"
 KAFKA_PORT="${CGW_KAFKA_PORT:-$DEFAULT_KAFKA_PORT}"
 DB_IP="${CGW_DB_IP:-$DEFAULT_DB_IP}"
 DB_PORT="${CGW_DB_PORT:-$DEFAULT_DB_PORT}"
-DB_USR="${CGW_DB_USER:-cgw}"
-DB_PASS="${CGW_DB_PASS:-123}"
+DB_USERNAME="${CGW_DB_USER:-cgw}"
+DB_PASSWORD="${CGW_DB_PASS:-123}"
 REDIS_DB_IP="${CGW_REDIS_DB_IP:-$DEFAULT_REDIS_DB_IP}"
 REDIS_DB_PORT="${CGW_REDIS_DB_PORT:-$DEFAULT_REDIS_DB_PORT}"
-LOG_LEVEL="${CGW_LOG_LEVEL:-info}"
 
 echo "Starting CGW..."
-echo "CGW ID: $ID"
-echo "LOG_LEVEL: $LOG_LEVEL"
-echo "GRPC: $GRPC_IP:$GRPC_PORT"
-echo "WSS: $WSS_LOCAL_IP:$WSS_LOCAL_PORT"
-echo "KAFKA: $KAFKA_IP:$KAFKA_PORT"
-echo "DB: $DB_IP:$DB_PORT"
-echo "REDIS_DB: $REDIS_DB_IP:$REDIS_DB_PORT"
-
-docker run -d -t --network=host --name $2 $1 ucentral-cgw \
-	--cgw-id $ID \
-	--grpc-ip $GRPC_IP --grpc-port $GRPC_PORT \
-	--wss-ip $WSS_LOCAL_IP --wss-port $WSS_LOCAL_PORT \
-	--kafka-ip $KAFKA_IP --kafka-port $KAFKA_PORT \
-	--db-ip $DB_IP --db-port $DB_PORT \
-	--db-username $DB_USR --db-password $DB_PASS \
-	--redis-db-ip $REDIS_DB_IP --redis-db-port $REDIS_DB_PORT \
-	$LOG_LEVEL
+echo "CGW LOG LEVEL: $LOG_LEVEL"
+echo "CGW ID       : $ID"
+echo "CGW WSS      : $WSS_LOCAL_IP:$WSS_LOCAL_PORT"
+echo "CGW GRPC     : $GRPC_IP:$GRPC_PORT"
+echo "CGW KAFKA    : $KAFKA_IP:$KAFKA_PORT"
+echo "CGW DB       : $DB_IP:$DB_PORT"
+echo "CGW REDIS    : $REDIS_DB_IP:$REDIS_DB_PORT"
+
+docker run \
+	-e CGW_LOG_LEVEL=$LOG_LEVEL      \
+	-e CGW_ID=$ID                    \
+	-e CGW_WSS_IP=$WSS_LOCAL_IP      \
+	-e CGW_WSS_PORT=$WSS_LOCAL_PORT  \
+	-e CGW_GRPC_IP=$GRPC_IP          \
+	-e CGW_GRPC_PORT=$GRPC_PORT      \
+	-e CGW_KAFKA_IP=$KAFKA_IP        \
+	-e CGW_KAFKA_PORT=$KAFKA_PORT    \
+	-e CGW_DB_IP=$DB_IP              \
+	-e CGW_DB_PORT=$DB_PORT          \
```

Comment:
I think it would be a good idea to allow user to pass other options here too, that are already configurable (code wise) in CGW, such as: Kafka Topic names, DB Table name

Reply:
id agree with paul, ive tested this so far and it is working as expected, just minor recommended changes.